openmc/plotter.py

```diff
@@ -53,16 +53,15 @@
 _MAX_E = 20.e6
 
 
-def plot_xs(this, types, divisor_types=None, temperature=294., data_type=None,
-            axis=None, sab_name=None, ce_cross_sections=None,
-            mg_cross_sections=None, enrichment=None, plot_CE=True, orders=None,
-            divisor_orders=None, **kwargs):
+def plot_xs(this, types, divisor_types=None, temperature=294., axis=None,
+            sab_name=None, ce_cross_sections=None, mg_cross_sections=None,
+            enrichment=None, plot_CE=True, orders=None, divisor_orders=None):
     """Creates a figure of continuous-energy cross sections for this item.
 
     Parameters
     ----------
-    this : str or openmc.Material
-        Object to source data from
+    this : {str, openmc.Nuclide, openmc.Element, openmc.Macroscopic, openmc.Material}
```

Comment:
openmc.Nuclide, openmc.Element, and openmc.Macroscopic are all effectively deprecated and were replaced by normal strings. We should really just get rid of them. So, I wouldn't mention them here.